gen/wrap_gdal.jl

```diff
@@ -5,7 +5,7 @@ workdir = dirname(@__FILE__)
 srcdir = joinpath(workdir, "../src")
 include(joinpath(workdir, "doc.jl"))
 
-gdalpath = "/usr/local/include"
+gdalpath = "/home/martijn/bin/gdal/include"
```

Comment:
Maybe leave a note in the `GDAL.jl generation documentation` for developers to update this `gdalpath` variable to their respective locations?